winrt/lib/text/CanvasFontFace.cpp

```diff
@@ -666,6 +666,114 @@ IFACEMETHODIMP CanvasFontFace::GetGlyphIndices(
         });
 }
 
+IFACEMETHODIMP CanvasFontFace::GetGlyphMetrics(
+  uint32_t inputCount,
+  int* inputElements,
+  boolean isSideways,
+  uint32_t* outputCount,
+  Rect** outputElements)
+{
+  return ExceptionBoundary(
+    [&]
+  {
+    CheckInPointer(inputElements);
+    CheckInPointer(outputCount);
+    CheckAndClearOutPointer(outputElements);
+
+    std::vector<unsigned short> glyphIndices;
+    glyphIndices.reserve(inputCount);
+    for (uint32_t i = 0; i < inputCount; ++i)
+    {
+      if (inputElements[i] < 0 || inputElements[i] > USHORT_MAX)
+        ThrowHR(E_INVALIDARG);
+
+      glyphIndices.push_back(static_cast<unsigned short>(inputElements[i]));
+    }
+
+    std::vector<DWRITE_GLYPH_METRICS> glyphMetrics(inputCount);
+    ThrowIfFailed(GetRealizedFontFace()->GetDesignGlyphMetrics(glyphIndices.data(), inputCount, glyphMetrics.data(), isSideways));
+
+    ComArray<Rect> output(inputCount);
+
+    DWRITE_FONT_METRICS1 metrics;
+    GetRealizedFontFace()->GetMetrics(&metrics);
+
+    for (uint32_t i = 0; i < inputCount; ++i)
+    {
+      {
```

Comment:
It looks like there's an unneeded block, here. Would it be possible to remove it?